eras/shelley-ma/test-suite/src/Test/Cardano/Ledger/ShelleyMA/Serialisation/Roundtrip.hs

```diff
@@ -2,31 +2,66 @@
 {-# LANGUAGE DataKinds #-}
 {-# LANGUAGE FlexibleContexts #-}
 {-# LANGUAGE MonoLocalBinds #-}
+{-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE TypeApplications #-}
 
 module Test.Cardano.Ledger.ShelleyMA.Serialisation.Roundtrip where
 
+-- import Cardano.Ledger.Compactible (Compactible(toCompact))
```

Comment:
There is some code commented out in this file in order to provide sufficient context for inspecting test failures.